packages/sitecore-jss-rendering-host/package.json

```diff
@@ -31,16 +31,18 @@
     "ngrok": "^3.2.5",
     "opn": "^6.0.0",
     "webpack": "4.41.0",
-    "webpack-dev-server": "^3.11.0"
+    "webpack-dev-server": "3.11.0"
```

Comment:
Why remove the `^`?

Reply:
Because when I did the migration to yarn we lost package.lock - I didn't have versions, so I had to fix some of them, cause it will make some errors